### ISSUE TITLE
Delete games

### ISSUE THREAD

**Issue by mikekasprzak** [ludumdare/ludumdare#1634]
Be able to delete games.

Due to how team-building works right now, users sometimes create games by accident. It would be nice if they could delete them.

**Comment by mikekasprzak** [comment]
Generally I get support emails from team members that accidentally created
their own game before being added to a game. So by delete I mean give
everyone a way to make a game go away.

The problem with actual deleting is blog posts are the child of the event
they we're written for, so deleting them orphans the posts.

**Comment by SnowTelevision** [comment]
I strongly support this!

**Comment by drschwabe** [comment]
yeah this is uninituitive, as a workaround I just create a new account (after haphazardly clicking "Create Game" when I was actually a team member needing to be added to an existing game)

**Comment by Hekateras** [comment]
Four years later and this is still an issue. Please fix!
